.CI/build_all

```diff
@@ -5,24 +5,36 @@
 export CONDA_NPY=19
 
 # Remove homebrew.
-brew remove --force --ignore-dependencies $(brew list)
-brew cleanup -s
-rm -rf $(brew --cache)
+echo ""
+echo "Removing homebrew from Travis CI to avoid conflicts."
+curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew.sh
```

Comment:
Missed the `.sh`. Fixed in PR ( https://github.com/conda-forge/staged-recipes/pull/2263 ).